adapters/consumable/consumable.go

```diff
@@ -32,6 +41,8 @@ type bidRequest struct {
 	Url                string      `json:"url,omitempty"`
 	EnableBotFiltering bool        `json:"enableBotFiltering,omitempty"`
 	Parallel           bool        `json:"parallel"`
+	Ccpa               string      `json:"ccpa,omitempty"`
```

Comment:
Nitpick: The standard naming in Go is to use uppercase letters for acronyms. Please consider `CCPA` and `GDPR`.